crates/holochain/src/conductor/conductor.rs

```diff
@@ -226,7 +226,7 @@ pub(crate) type StopReceiver = task_motel::StopListener;
 /// A Conductor is a group of [Cell]s
 pub struct Conductor {
     /// The collection of available, running cells associated with this Conductor
-    running_cells: RwShare<HashMap<CellId, CellItem>>,
+    running_cells: RwShare<IndexMap<CellId, CellItem>>,
```

Comment:
This is to ensure consistent ordering of the serialized response, to allow a test to assert that the response matches an expected value.